tests/drivers/build_all/display/app.overlay

```diff
@@ -23,6 +23,13 @@
 			status = "okay";
 		};
 
+		test_pwm1: pwm@55551111 {
+			compatible = "vnd,pwm";
+			#pwm-cells = <3>;
+			reg = < 0x55551111 0x1000 >;
+			status = "okay";
+		};
+
```

Comment:
drop?

Reply:
Thank you @kartben

Reply:
Fixed in [810703f](https://github.com/zephyrproject-rtos/zephyr/commit/810703f196aaa5242daf43eb983aeca14d0fb18b)